### DISCUSSION TITLE
Modal issues in 5.2.1 version

### DISCUSSION THREAD

**Discussion by filipbekic01** [twbs/bootstrap#37113] [Q&A] [answered]
All of the sudden, after updating to 5.2.1 our modals started behaving weirdly. Whenever we try to focus input modals disappear. 

Anybody else experience same thing?

**Comment by abbey4010** [comment]
Good morning i haven't  though just started using it if you can  teach me please

**Reply by filipbekic01** [reply]
I tried really hard to understand your sentence but it absolutely makes no sense.

**Comment by julien-deramond** [comment, marked as answer]
@filipbekic01 I tried a simple example with the [Live demo](https://getbootstrap.com/docs/5.2/components/modal/#live-demo) containing an input within the `.modal-body`; the focus state seems to be OK and doesn't make the modal disappear.
Would you have a minimalist example to share to be able to reproduce this behaviour?

**Reply by filipbekic01** [reply]
Finally got time to create replica https://codesandbox.io/s/hopeful-cherry-uez1u9 here. Note that I'm using Vue 3 here, I'm not sure if I'm reporting this at right repository tho, let me know if I'm wrong and whom should I contact about this. Here are steps to reproduce the bug:

1. Open modal
2. Click green button once
3. Click red button once

Reload page to test it again.

I find it funny because bug only happens if I have v-if conditional in vuejs. Otherwise it works good. Any suggestions?

Thanks in advance.

**Reply by filipbekic01** [reply]
Any help would be appreciated :)

**Reply by GeoSot** [reply]
Probably because of this https://github.com/twbs/bootstrap/pull/37128?

You can try to use this script https://deploy-preview-37128--twbs-bootstrap.netlify.app/docs/5.2/dist/js/bootstrap.bundle.min.js just to be sure

**Reply by filipbekic01** [reply]
That might help, I'll just wait for 5.2.2 release. Thanks @GeoSot !

**Comment by abbey4010** [comment]
I said can you like give me hints on how to use it
